org.purei.Play.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: org.purei.Play
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: Play
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host
```

Comment:
that's a very broad filesystem permission

Reply:
i was meant to make this `--filesystem=host:ro` if you have different suggestion, I'd like to hear it.
my decision was based on fact that the user could have their games stored anywhere including external storage.

Reply:
is this better?

Reply:
I'd say just `host:ro` at this point. The most sensitive data, which is likely in the home directory, can already be read.

The permissions will be simpler, and people can always restrict more/change to `home` later with e.g. Flatseal.

Reply:
for removable storage devices there is `/run/media` and `/media`.

Reply:
`/mnt` makes sense since some people (like me) mount disks permanently there